lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -737,6 +740,28 @@ fromNonMyopicMemberRewards =
     . Map.mapKeys (bimap fromShelleyCoin fromStakeCredential)
     . O.unNonMyopicMemberRewards
 
+getDesirabilities
+    :: forall crypto. ()
+    => SL.RewardProvenance crypto
+    -> Map W.PoolId W.StakePoolDesirability
+getDesirabilities =
+    Map.map fromDesirability
+    . Map.mapKeys fromPoolId
+    . SL.desirabilities
+    where
+    fromDesirability (SL.Desirability x y) = W.StakePoolDesirability x y
```

Comment:
Hi @HeinrichApfelmus 

Just a tiny point of style! We usually deindent `where` by two spaces, to make it stand out visually. So like this:

```suggestion
getDesirabilities =
    Map.map fromDesirability
    . Map.mapKeys fromPoolId
    . SL.desirabilities
  where
    fromDesirability (SL.Desirability x y) = W.StakePoolDesirability x y
```